extensions/curator/curator-docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+
+services:
+
+
+  curator:
+    build:
+      context: extensions/curator/
+    environment:
+      ELASTICSEARCH_HOST: "elasticsearch"
+      CRON: "0 0 * * *"
+      CONFIG_FILE: "/usr/share/curator/config/curator.yml"
```

Comment:
Would relative paths work since you changed the container's workdir in the Dockerfile?